sdk/identity/azure-identity/README.md

```diff
@@ -88,7 +88,7 @@ The `DefaultAzureCredential` is appropriate for most scenarios where the applica
  - Environment - The `DefaultAzureCredential` will read account information specified via [environment variables](#environment-variables) and use it to authenticate.
  - Managed Identity - If the application is deployed to an Azure host with Managed Identity enabled, the `DefaultAzureCredential` will authenticate with that account.
  - IntelliJ - If the developer has authenticated via Azure Toolkit for IntelliJ, the `DefaultAzureCredential` will authenticate with that account.
- - Visual Studio Code - If the developer has authenticated via the Visual Studio Code Azure Account plugin, the `DefaultAzureCredential` will authenticate with that account.
+ - Visual Studio Code - If the developer has authenticated via the Visual Studio Code Azure Account extension (**version 0.9.11 or earlier**), the `DefaultAzureCredential` will authenticate with that account.
```

Comment:
Should we be doing the "Known issues" section for beta.2 release or for GA as well?

Reply:
Should the "Known issues" section appear at the top of the change log? Seems that's best since the issue isn't really tied to a particular version of the Azure Identity library.

Reply:
I think we should confirm with @weshaggard @chidozieononiwu

Reply:
I don't think we have any defined section in the README for this so you can do what you feel is best.